doc/source/user_guide/visualization.rst

```diff
@@ -1330,6 +1330,13 @@ When multiple axes are passed via the ``ax`` keyword, ``layout``, ``sharex`` and
 don't affect to the output. You should explicitly pass ``sharex=False`` and ``sharey=False``,
 otherwise you will see a warning.
 
+.. ipython:: python
+   :suppress:
+
+   import matplotlib
+
+   matplotlib.rc('xtick', labelsize=8)
+
```

Comment:
I'm not sure this should be hidden - if someone wants to reproduce the plot and it looks different, that might be confusing. Would simply increasing the `figsize` in `plt.subplots` be enough to make it more legible? Otherwise, can `matplotlib.rc('xtick', labelsize=8)` be made visible to the readers?

Reply:
You're right, I just did not want to mess up the code snippet... but increasing fig size is equally fine!